circle.yml

```diff
@@ -5,13 +5,22 @@ machine:
   node:
     version:
       6.9.1
+  environment:
+    FIREFOX_VERSION: "59.0"
+    GECKODRIVER_VERSION: "0.20.0"
+    MOZ_HEADLESS: 1
 dependencies:
   override:
     - pip install --upgrade setuptools
     - pip install -r requirements_dev.txt
     - pip install -r requirements.txt
     - npm install
     - npm run build
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install --no-install-recommends -y bzip2 curl firefox
+    - sudo apt-get -y purge firefox
```

Comment:
Why does this install Firefox here using apt first and then installs it again from the CDN?

Reply:
As I understand, installing Firefox via apt-get creates certain folders that are required for a proper installation. Any thoughts @davehunt?